streams/src/test/java/org/apache/kafka/streams/processor/internals/StreamTaskTest.java

```diff
@@ -186,19 +197,11 @@ public void cleanup() throws IOException {
     @SuppressWarnings("unchecked")
     @Test
     public void testProcessOrder() {
-        task = createStatelessTask(false);
+        task = createStatelessTask(createConfig());
 
-        task.addRecords(partition1, records(
-                new ConsumerRecord<>(partition1.topic(), partition1.partition(), 10, 0L, TimestampType.CREATE_TIME, 0L, 0, 0, recordKey, recordValue),
-                new ConsumerRecord<>(partition1.topic(), partition1.partition(), 20, 0L, TimestampType.CREATE_TIME, 0L, 0, 0, recordKey, recordValue),
-                new ConsumerRecord<>(partition1.topic(), partition1.partition(), 30, 0L, TimestampType.CREATE_TIME, 0L, 0, 0, recordKey, recordValue)
-        ));
+        task.addRecords(partition1, Arrays.asList(new ConsumerRecord<>(partition1.topic(), partition1.partition(), 10, 0L, TimestampType.CREATE_TIME, 0L, 0, 0, recordKey, recordValue), new ConsumerRecord<>(partition1.topic(), partition1.partition(), 20, 0L, TimestampType.CREATE_TIME, 0L, 0, 0, recordKey, recordValue), new ConsumerRecord<>(partition1.topic(), partition1.partition(), 30, 0L, TimestampType.CREATE_TIME, 0L, 0, 0, recordKey, recordValue)));
```

Comment:
nit: flattening to a very long single line, is it intentional?

Reply:
Ditto below and in other tests like `testPauseResume`

Reply:
it was not. I've fixed it.